chain/network/src/peer.rs

```diff
@@ -632,14 +634,22 @@ impl StreamHandler<Result<Vec<u8>, ReasonForBan>> for Peer {
             msg.len() as i64,
         );
 
+        // TODO(HandshakeV2): Remove this, and deprecate PeerMessage::Handshake
```

Comment:
If the plan is to keep this TODO, it should be in the format "TODO(#\<tracking issue\>):"

Reply:
Addressed in #3157